api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -48,17 +47,16 @@ message VmConfig {
 
 // Base Configuration for Wasm Plugins e.g. filters and services.
 // [#next-free-field: 6]
-// [#not-implemented-hide:] pending implementation.
 message PluginConfig {
   // A unique name for a filters/services in a VM for use in identifying the filter/service if
-  // multiple filters/services are handled by the same *vm_id* and *group_name* and for
+  // multiple filters/services are handled by the same *vm_id* and *root_id* and for
   // logging/debugging.
   string name = 1;
 
   // A unique ID for a set of filters/services in a VM which will share a RootContext and Contexts
   // if applicable (e.g. an Wasm HttpFilter and an Wasm AccessLog). If left blank, all
-  // filters/services with a blank group_name with the same *vm_id* will share Context(s).
-  string group_name = 2;
+  // filters/services with a blank root_id with the same *vm_id* will share Context(s).
+  string root_id = 2;
```

Comment:
Please add the `option (udpa.annotations.file_status).work_in_progress = true;` annotation here. xDS style does not allow breaking changes to any package in the v3 namespace unless this is set.

Reply:
I added option (udpa.annotations.file_status).work_in_progress = true; and format fix removed it.

Any suggestions on how to get the "./tools/proto_format/proto_format.sh fix" to leave the annotation?

Reply:
Ok, I think that things need to be marked "alpha" otherwise the work_in_progress flag is removed (just guessing here, I can't find a reference in the docs).

However, the previous version was marked as #not-implemented-hide.

Does that still mean that we can't change it  because of xDS style?

Reply:
Let me add that "group_name" has never been used in code or configuration anywhere at any time.   The concept of a root and a root_id are deeply ingrained over 3 repos and many lines of code.  The code in envoyproxy/envoy is vestigial at best and while I understand the goals of the style, in practice envoyproxy/envoy-wasm has been the source of truth for these protos and what we are trying to do here is to make envoyproxy/envoy the source of truth.  In that sense we are correcting a bug, not breaking with a "standard".

Reply:
I think it's fine as absolutely nobody should be using this, but I would appreciate if you added the WiP annotation. to make this clear.

Reply:
The annotations seem to work fine in `api/envoy/extensions/filters/network/postgres_proxy/v3alpha/postgres_proxy.proto`. Can you share with me a snapshot of your tree with the annotation and I can try and debug?

Reply:
https://github.com/envoyproxy/envoy/pull/12257

Reply:
format fix removes the annotations for me locally.

Reply:
As you can see here : https://dev.azure.com/cncf/4684fb3d-0389-4e0b-8251-221942316e06/_apis/build/builds/45954/logs/12